pom.xml

```diff
@@ -91,6 +87,40 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <maven.min-version>3.5.0</maven.min-version>
+    <plugin.build-helper-maven.version>3.3.0</plugin.build-helper-maven.version>
+    <plugin.cyclonedx-maven.version>2.7.9</plugin.cyclonedx-maven.version>
+    <plugin.duplicate-finder-maven.version>1.5.1</plugin.duplicate-finder-maven.version>
+    <plugin.exec-maven.version>3.1.0</plugin.exec-maven.version>
+    <plugin.formatter-maven.version>2.23.0</plugin.formatter-maven.version>
+    <plugin.git-commit-id-maven.version>4.9.9</plugin.git-commit-id-maven.version>
+    <plugin.impsort-maven.version>1.8.0</plugin.impsort-maven.version>
+    <plugin.jacoco-maven.version>0.8.8</plugin.jacoco-maven.version>
+    <plugin.maven-assembly.version>3.4.2</plugin.maven-assembly.version>
+    <plugin.maven-checkstyle.version>3.2.1</plugin.maven-checkstyle.version>
+    <plugin.maven-clean.version>3.2.0</plugin.maven-clean.version>
+    <plugin.maven-compiler.version>3.10.1</plugin.maven-compiler.version>
+    <plugin.maven-dependency.version>3.5.0</plugin.maven-dependency.version>
+    <plugin.maven-deploy.version>3.1.0</plugin.maven-deploy.version>
+    <plugin.maven-enforcer.version>3.2.1</plugin.maven-enforcer.version>
+    <plugin.maven-gpg.version>3.1.0</plugin.maven-gpg.version>
+    <plugin.maven-install.version>3.1.0</plugin.maven-install.version>
+    <plugin.maven-jar.version>3.3.0</plugin.maven-jar.version>
+    <plugin.maven-javadoc.version>3.4.1</plugin.maven-javadoc.version>
+    <plugin.maven-jxr.version>3.3.0</plugin.maven-jxr.version>
+    <plugin.maven-pmd.version>3.20.0</plugin.maven-pmd.version>
+    <plugin.maven-project-info-reports.version>3.4.2</plugin.maven-project-info-reports.version>
+    <plugin.maven-release.version>3.0.0</plugin.maven-release.version>
+    <plugin.maven-replacer.version>1.5.3</plugin.maven-replacer.version>
+    <plugin.maven-resources.version>3.3.0</plugin.maven-resources.version>
+    <plugin.maven-scm-publish.version>3.2.1</plugin.maven-scm-publish.version>
+    <plugin.maven-site.version>3.12.1</plugin.maven-site.version>
+    <plugin.maven-source.version>3.2.1</plugin.maven-source.version>
+    <!-- seems to have an impact with java 11 integration, update with caution -->
```

Comment:
Do we think this comment is still relevant?
Maybe addressed in a future PR if we upgrade to 3.5.0 or 3.1.2

Reply:
Yeah, there are a few things that need addressing like that.